pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -633,12 +633,13 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd *OSDInfo, provision
 			Labels: osdLabels,
 		},
 		Spec: v1.PodSpec{
+			HostNetwork:        c.spec.Network.IsHost(),
```

Comment:
Can you revert this file? It seems nothing really changed here.

Reply:
done